### ISSUE TITLE
[UX]: Improve tokens list table (scrollbar issue)

### ISSUE THREAD

**Issue by buberdds** [oasisprotocol/explorer#1591]
### IMPACT

- [ ] High
- [ ] Medium
- [X] Low

### URGENCY

- [ ] Critical
- [ ] Major
- [ ] Medium
- [X] Minor

### DESCRIPTION OF ISSUE

There is a large horizontal scrollbar (test resulution 1250px). We need to improve tokens list table so user can see columns without using a scrollbars. I am not sure all requested changes are applied + we are adding "on-chain market cap" column.

### DEVICE / VIEWPORT

- [ ] Desktop
- [ ] Mobile

**Comment by tjanez** [comment]
I think this was the issue on my end with #1506 (URL: https://pr-1506.oasis-explorer.pages.dev/mainnet/sapphire/token):

![Image](https://github.com/user-attachments/assets/02edd31c-3d48-4e88-b04a-60fd1463bcf5)

**Comment by tjanez** [comment]
I agree with using "minimized" versions of columns in case there is less horizontal space available (e.g. in split screen use cases).
